packages/docusaurus-plugin-content-blog/src/feed.ts

```diff
@@ -79,6 +69,26 @@ async function generateBlogFeed({
       },
     } = post;
 
+    const potentialPaths = [
+      path.join(
+        outDir,
+        permalink.replace(siteConfig.baseUrl, ''),
+        'index.html',
+      ),
+      path.join(outDir, `${permalink.replace(siteConfig.baseUrl, '')}.html`),
+    ];
```

Comment:
that seems like a useful function that we could eventually move to docusaurus utils?

What about extracting it?

Also maybe change the ordering of paths according to the trailingSlash option?

If trailingSlash is false, do we really want to lookup /path/index.html?